packages/astro/src/content/utils.ts

```diff
@@ -195,6 +205,14 @@ export async function getEntryData(
 			throw formattedError;
 		}
 	}
+	if (imageImports.size > 0) {
+		Object.defineProperties(data, {
+			_internal_imageImports: {
```

Comment:
Bit of a hack, admittedly, but it avoids needing to change the usage of `getEntryData`

Reply:
Can you leave a comment that explains the hack and why?

Reply:
Looking at the hack again made me sad, so I've removed it and made a new method.